config/locales/en.yml

```diff
@@ -641,6 +641,7 @@ en:
         case_type:
           income_tax_html: "<strong>Income Tax</strong><br>including tax paid through
```

Comment:
Nothing to do with this PR, but these hard-coded <strong> tags seem unnecessary.